0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Interledger Protocol V4 (ILPv4)
+draft: 1
+---
+
+# Interledger Protocol V4
+
+Interledger is a protocol for sending packets of money across different payment networks or ledgers. ILPv4 is a simplification of previous versions of the protocol designed primarily for "penny switching", or routing large volumes of smaller-value packets. ILPv4 can be integrated with any type of ledger, including those not built for interoperability, and it can be used with a variety of higher-level protocols that implement features ranging from quoting to sending larger amounts of value with chunked payments. 
+
+## Overview
+
+### Design Goals
+
+- **Neutrality** - Interledger is not tied to any company, currency, or network.
+- **Interoperability** - ILP should be usable across any type of ledger, even those that were not built for interoperability.
+- **Security** - Senders, connectors, and receivers should be protected from one another and especially isolated from risks posed by parties that they are indirectly connected to. Connectors should not be able to steal money from senders, and senders should not be able to tie up too much of connectors' funds or otherwise interfere with their operation.
```

Comment:
`tie up too much of connectors' funds` -> `tie up connectors' funds`

Reply:
Why? You tie up connectors' funds every time you prepare a payment. The problem is if you take up too much bandwidth